src/Time/Units.hs

```diff
@@ -163,6 +169,15 @@ instance NFData (Time (rat :: Rat))
 instance Serialise (Time (rat :: Rat))
 #endif
 
+#ifdef HAS_aeson
+instance (KnownUnitName unit) => ToJSON (Time (unit :: Rat)) where
+    toJSON = toJSON . show
```

Comment:
I like the default behavior. But let's also implement `newtype NatTime unit` with JSON instances like in `yt-utilities`. I think its quite useful in many cases.

Reply:
But that's going to be not type safe instance then :thinking:  I mean `time: 5` can be minute, second, etc. and that could lead to some issues. The one that I've implemented uses `read` that's why it would work as I expect it to be..

Reply:
@vrom911 Hmm, you're right. We don't want to motivate unsafe behavior. I guess if people work with unsafe API they should do some tricks to work carefully. Changing type of unit shouldn't go without problems. So, if thinking more about `NatTime` newtype it probably shouldn't exist.

Reply:
Okay :+1: Thanks for clarifications!